src/ConfigData/TableConfigurations/TableConfiguration.cs

```diff
@@ -148,8 +149,9 @@ public string TableName
             }
         }
 
-        public void ReadFromJSON(string JSONstring)
+        public string ReadFromJSON(string JSONstring, int hwnd)
```

Comment:
no, a HWND can not be required just to read a JSON
Maybe this can be optional but even that this is not the right place